test/plugin/test_out_elasticsearch.rb

```diff
@@ -954,10 +1001,10 @@ def test_connection_failed_retry
       raise Faraday::ConnectionFailed, "Test message"
     end
 
-    driver.emit(sample_record)
-
-    assert_raise(Fluent::ElasticsearchOutput::ConnectionFailure) {
-      driver.run
+    assert_raise(Fluent::Plugin::ElasticsearchOutput::ConnectionFailure) {
```

Comment:
In v0.14, driver#feed does not raise Error.
We can remove this assertion safely.